target/linux/ath79/dts/ar9342_zyxel_nwa112x.dtsi

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	aliases {
+		label-mac-device = &eth0;
+
```

Comment:
Omit line break.

Reply:
A significant majority of devices indeed don't have a newline here, so I'll omit this one. There seems to be no consensus however on whether label-mac-device should go to the top or bottom of aliases.